src/main/resources/application.yml

```
@@ -0,0 +1,22 @@
+spring:
+    datasource:
+        driver-class-name:
+        url: ${DB_URL}
+        username: ${DB_USERNAME}
+        password: ${DB_PASSWORD}
+jpa:
+    hibernate:
+        ddl-auto: update
+    properties:
+        hibernate:
+            dialect: org.hibernate.dialect.MySQLDialect
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)